README.md

```diff
@@ -1 +1,133 @@
-# Awesome-books-with-ES6
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [Awesome-books-with-ES6] <a name="about-project"></a>
+
+> **[Awesome Books]** is a...This is a simple html page which is contains a dynamic form about Awesome books, books title, and the author. I have used HTML and Javascript for this project and you can use it in your local machine just by cloning this repo.
+
```

Comment:
- [ OTPIONAL] Kindly edit this document to remove all the `>` marks. These mark out the content as a guide. 
![guide](https://user-images.githubusercontent.com/99656687/235423051-933f233f-7db9-4f34-a59d-765307df86c6.png)